ui/src/components/DashboardGlobalHealth.js

```diff
@@ -19,7 +19,9 @@ import { useStartingTimeStamp } from '../containers/StartTimeProvider';
 import LoaderComponent from '@scality/core-ui/dist/components/loader/Loader.component';
 import CircleStatus, { StatusIcon } from './CircleStatus';
 import GlobalHealthBarComponent from '@scality/core-ui/dist/components/globalhealthbar/GlobalHealthBar.component';
-import { useHistoryAlerts } from '../containers/AlertHistoryProvider';
+import { getClusterAlertSegement } from '../services/platformlibrary/metrics';
```

Comment:
Small typos and I would rename it as follows
```suggestion
import { getClusterAlertSegmentQuery } from '../services/platformlibrary/metrics';
```